client/components/common/mixins/publish.vue

```diff
@@ -9,6 +9,7 @@ const prefix = message => `Are you sure you want to publish ${message}`;
 export default {
   data() {
     return {
+      activityNum: 0,
       isPublishing: false,
```

Comment:
`isPublishing` is merely an computed value calculated from `publishStatus`, isn't it?
Personally it makes more sense to me to use something like:
```js
data() {
  return {
    publishStatus: {
      progress: 0,
      message: ''
    }
  }
},
computed() {
  isPublishing: ({ publishStatus }) => publishStatus.progress > 0;
}
```